code/game/machinery/mindswapper.dm

```diff
@@ -13,6 +13,7 @@
 	var/operating = FALSE  // Is it on?
 	var/swap_time = 200  // Time from starting until minds are swapped
 	var/swap_range = 1
+	var/list/swap_blacklist = list(/mob/living/simple_animal/hostile/megafauna)
```

Comment:
No fun.

Reply:
This mob never exists, the blacklist exists to block players from swapping into mobs that are just outright broken. It doesn't use istype so only this specific one is blocked, the others like the hivemind megafauna are blocked for control spiders as well.

Reply:
It should work that way at least.

Reply:
Control spiders check a specific variable to see if they are swapping into a robotics mob, so no robotic mobs work, but if it's not an istype check, it's fine

Reply:
This uses the same check, MOB_CLASSIFICATION to make sure the mob isn't synthetic.